### PR TITLE
Update wording of monorepo README and update to Polywrap brand

### PR DESCRIPTION
**Pull request by kev-ngo**

### CODE REVIEW

**Review by cbrzn — DISMISSED** [review]
LGTM 🍏

File: README.md [hunk 1]
```diff
@@ -1,47 +1,96 @@
...
+To lint your project directory, run the following:
+
+```
+yarn lint
+```
+
+To auto-fix lint errors:
+
+```
+yarn lint: fix
```
Comment (Niraj-Kamdar) [inline README.md:81]:
```suggest
yarn lint:fix
```

**Review by Niraj-Kamdar — CHANGES_REQUESTED** [review]
There is a minor typo

**Review by dOrgJelli — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (dOrgJelli) [reply README.md:81]: Shoot, missed this, will fix in prealpha-dev. Nice catch!

**Pull request merged**